src/fsd/4-entities/mow/mows.service.ts

```
@@ -0,0 +1,67 @@
+    private static getUpgrades(
+        upgrades: IMowLevelUpgrades | undefined,
+        key: 'primary' | 'secondary'
+    ): Array<IBaseUpgrade | ICraftedUpgrade> {
+        if (!upgrades) {
+            return [];
+        }
+        const rawUpgrades = upgrades[key] ?? upgrades.primary;
+
+        return rawUpgrades.map(upgrade => UpgradesService.getUpgrade(upgrade)).filter(x => !!x);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)